### ISSUE TITLE
Not compatible with Rails 5.2

### ISSUE THREAD

**Issue by bdavidxyz** [turbolinks/turbolinks#388]
Just rollbacked my app to Rails 5.1, now Turbolinks works like a charm. Need a separate app to demonstrate and investigate though. Anyone experienced the same problem ?

**Comment by dukex** [comment]
Hi @bdavidxyz I'm using Turbolinks 5.1.1 with Rails 5.2 and seems it's working well, what kind of issues are you having?

**Comment by bdavidxyz** [comment]
I tried to isolate the problem. Ruby version is 2.5.0. I created a github repo for turbolinks 5.1.1 + Rails 5.1 and the associated heroku app:

- https://github.com/bdavidxyz/turbolinks-rails-5-1-0
- https://turbolinks-rails-5-1-0.herokuapp.com/

Then I cloned this repository and only changed Rails from 5.1 to 5.2

- https://github.com/bdavidxyz/turbolinks-rails-5-2-0
- https://turbolinks-rails-5-2-0.herokuapp.com/

Both apps works properly. Admin can close this issue because I cannot duplicate and explain the original problem, despite isolation attempt.
